### PR TITLE
[Doc] [NextJS] Add procedures for getting-started/walkthrough-jsscreate

### PR DESCRIPTION
**Pull request by AncaIO**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:

- [ ] I have read the Contributing guide.
- [ ] My code follows the code style of this project.
- [ ] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change and it requires an update to the documentation.
- [ ] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

**Comment by AncaIO** [comment]
@nickwesselman  I took "procedural" quite literally. I hope that's ok. If we have anything to say about the CLI we have https://jss.sitecore.com/docs/fundamentals/cli.

File: docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md [hunk 1]
```diff
@@ -3,4 +3,67 @@ name: walkthrough-jsscreate
...
\ No newline at end of file
+# Walkthrough: Using the JSS CLI to get started with JSS and Next.js 
+
+You can create a JSS Next.js application by using the JSS command-line tool (CLI). 
+
+> When using this approach, you will need to perform additional steps to connect to a Sitecore instance.
+
+We recommend this approach for Sitecore developers that use the [code-first developer workflow](https://jss.sitecore.com/docs/fundamentals/dev-workflows/code-first). 
+
+## TL;DR
```
Comment (nickwesselman) [inline docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md:14]: I love it :)

File: docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md [hunk 2]
```diff
@@ -3,4 +3,67 @@ name: walkthrough-jsscreate
...
+
+1. In the terminal, go to your application directory. For example: `cd my-first-jss-app`.
+
+2. Run `jss start`.
+
+   > To see the full list of commands available for your application, run `jss --help`.
+
+## Connect to Sitecore
+
+Pulling content in your JSS application from Sitecore  and using JSS in production, will require a Sitecore instance.
```
Comment (nickwesselman) [inline docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md:59]: "Pulling" isn't really accurate, maybe something like

> Deploying your code-first artifacts and content to Sitecore, WYSIWYG editing, and use of JSS in production will require a Sitecore instance.

File: docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md [hunk 3]
```diff
@@ -3,4 +3,67 @@ name: walkthrough-jsscreate
...
+
+## Connect to Sitecore
+
+Pulling content in your JSS application from Sitecore  and using JSS in production, will require a Sitecore instance. 
+
+Sitecore requires Windows, but the instance can be on a virtual machine or a remote server. 
+
+To connect your application to Sitecore, you must:
+
+1. [Setup JSS Server Components](https://jss.sitecore.com/docs/getting-started/jss-server-install).
```
Comment (nickwesselman) [inline docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md:65]: Yeah I think linking to these for now is good, once we restructure all the docs to focus on Next.js, we can move / reword these articles as well.

File: docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md [hunk 4]
```diff
@@ -3,4 +3,67 @@ name: walkthrough-jsscreate
...
 ---
-# Walkthrough: with `jss create`
\ No newline at end of file
+# Walkthrough: Using the JSS CLI to get started with JSS and Next.js 
+
+You can create a JSS Next.js application by using the JSS command-line tool (CLI). 
+
+> When using this approach, you will need to perform additional steps to connect to a Sitecore instance.
+
+We recommend this approach for Sitecore developers that use the [code-first developer workflow](https://jss.sitecore.com/docs/fundamentals/dev-workflows/code-first).
```
Comment (nickwesselman) [inline docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md:12]: "or who wish to evaluate the JSS developer experience without installing Sitecore" ?

**Review by nickwesselman — CHANGES_REQUESTED** [review]

**Comment by nickwesselman** [comment]
> @nickwesselman I took "procedural" quite literally. I hope that's ok. If we have anything to say about the CLI, we have https://jss.sitecore.com/docs/fundamentals/cli.

Straightforward and simple, I like it.

**Review by nickwesselman — APPROVED** [review]

**Pull request merged**